examples/test/post-processing/index.html

```diff
@@ -0,0 +1,309 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Post Process</title>
+    <meta name="description" content="Post process — A-Frame">
+    <script src="../../../dist/aframe.js"></script>
+    
+    <script>
+        AFRAME.registerComponent('pulse', {
```

Comment:
Let's register these components in separate files: e.g., `components/pulse-effect.js`, `components/envmap.js`, `components/bloom.js`